src/drawing/DrawingCreator.scss

```diff
@@ -1,3 +1,12 @@
+$drawing_cursor_32: 'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAYAAAEEfUpiAAAABGdBTUEAALGPC/xhBQAAADhlWElmTU0AKgAAAAgAAYdpAAQAAAABAAAAGgAAAAAAAqACAAQAAAABAAAAIKADAAQAAAABAAAAIAAAAACPTkDJAAAECElEQVRYCe1WS0sbURQ2DxMTU0VpWtuCoQtraFXcCN1FuqqrrppNKNRVKdLSlZAfILh0qYhbuxPcFS2CBAkmxFdf4CMRsYuCLSURjXmYft+QM0zGmVFbV8UDZ+6de97nnnvvqavTgU3z78LcZq8uuDGecEH5Hx4e7sTEo/yYfpaXl+MkKjrcbvdJoVB4L9xUFABqLQpNHU2JLsCJ0+ksRKPRB1ottmw2uw5iPhaLvQWBLp7RwgU6dYaAtcuDqkVyJSps+/v7L2ZmZoJY0NPqbDs7O4Pz8/MrdBYMzLYKtkwm8zKVSq3X19cz67eBNRoaKoAq8ZaeSDUNAwMDDzHeNCKS4WpzQI0Caj5kwWAkj3hQwfy0ysO5JVDItbCwEGpqaspiXunp6Un39/czTifQ1LgiyJyzEJlaCqNC1xYXF19h3mKmgILOra2tMEtuZGQkRUGmvlphTD+rzLSgqMA7NzcXh6t7bW1tv5aWlt5gjYINZoLaODindpaIA8hkFYFlIBN2btLA8z8D86PNlxork2UFIki+muNlJSQ02RVvLpf7ls/nWRdeIJUZeiOCHMlARl+pVNrFqGzj0dHRKuZSE5gauyWWPel0OoZLsP0rgMzFYrERA8+BKaiWIbw6NTWVtNvtJXCfIoQ9jPeADMMwH2bCFZyPDQgFgM1A0xxQgQeX2wrGStWyXtjyGFNzS3d3914wGPxOJZubm58wtgNpuR5II6bgxHl/AmoFSvZ1bltaFo30oBm3+lOM94F3gTeAlsJalzinm+Iqs8/jzGNtepT1CsCrxilCMpJ2DVefAe0e/K12vQ7+6/dN/6/aMjyTKtV6QkOCLFXX7Ozsne3t7ceRSITly8eF67RBFF5M/w1EEZXSiA+PcIRXAN5QNj7KC354eLgxPT39CP+8wXg05A6hvAo1P+qq8UR4aZjKXDD83OFwvDs4OCj5fD5XV1dXJ+5fGlMgHo8n0KJGQ6FQAgsFIM8mt8N0S0A7AxKxYhRURjzI65atx9jYWLKvry+D9VM0Bau4ur+gl8lhKzaSySRbET9lgHRMtgLT80Fr2A32GsO869n3yK0LutLGwLnP4+Pjz/DPS5RdidSD4RaAbgg0Tm/pdSP7V4l4cnIyYWSYl2+1hwpAhj0ub3D2UlKIEhCWrEGMs8BuHB8ff0Cf9RMRJ5DyY6zJHioR88nQGGbETUAa1qacOi8MZKbX3qGhoQCiX+vt7d0tl8t5NuoTExMpdo+6iGmYEfPBv1TEZDYFRO70+/2e0dHRH3Ak19raaguHw787OjpeAz9CkMeOKBWufXkuVOlGqeGaZIFpJNJRFhGV0giPkzx37F4vbRgyChg5QIKsc9QfHbUGQNNGqZ1TxzVcKAN/AHl7eTVxfPPmAAAAAElFTkSuQmCC';
```

Comment:
We should probably rename all of these cursor variables to match our conventions. Maybe `$ba-DrawingCreator-cursor`?